tests/tests/parsers/code_2e10_wip.log

```diff
@@ -8,3 +8,7 @@
 2023-11-04T17:38:34.781338 067  I 024 21:052764 --:------ 21:052764 2E10 002 0001  # {'presence_detected': True, '_unknown_4': '', 'seqx_num': '024'}
 2023-11-04T17:38:34.867259 067  I 024 21:052764 --:------ 21:052764 2E10 002 0001  # {'presence_detected': True, '_unknown_4': '', 'seqx_num': '024'}
 2023-11-04T17:38:34.942291 068  I 024 21:052764 --:------ 21:052764 2E10 002 0001  # {'presence_detected': True, '_unknown_4': '', 'seqx_num': '024'}
+
+# ClimaRad VenturaV1x reporting, no option for presence detection, eg. just 1 change in 24 h @ 7PM
```

Comment:
Is this 7pm feature configurable?

Reply:
I do not know as the docs don't say. Once we have this visible in HA I can keep an eye.